From 19b212de3a6979e877e61f6a7c3d81570a698738 Mon Sep 17 00:00:00 2001
From: Eliad Peller <eliad@wizery.com>
Date: Wed, 18 May 2011 18:55:15 +0300
Subject: [PATCH 4/9] wl12xx: update wl12xx_platform_data struct

(without this patch both the kernel and compat compilations
will succeed, but the wl12xx driver will use uninitialized
memory for the platform quirks...)

Signed-off-by: Eliad Peller <eliad@wizery.com>
---
 include/linux/wl12xx.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/include/linux/wl12xx.h b/include/linux/wl12xx.h
index cc2a6e1..19c7a2c 100644
--- a/include/linux/wl12xx.h
+++ b/include/linux/wl12xx.h
@@ -39,6 +39,7 @@ struct wl12xx_platform_data {
 	bool use_eeprom;
 	int board_ref_clock;
 	int board_tcxo_clock;
+	unsigned long platform_quirks;
 };
 
 #ifdef CONFIG_WL12XX_PLATFORM_DATA
-- 
1.7.0.4

